src/System.Data.Common/ref/project.json

```diff
@@ -5,9 +5,9 @@
     "System.IO": "4.0.0"
   },
   "frameworks": {
-    "netstandard1.2": {
+    "netstandard1.0": {
```

Comment:
You could have fixed this the other way.  The ref itself actually packages as netstandard1.2.  I'm ok as is, just something we'll have to address when collapsing NuGetTargetMoniker and PackageTargetFramework in the future.